### DISCUSSION TITLE
[Feature Request]: Remote wake GPS/ request position

### DISCUSSION THREAD

**Discussion by samuk** [meshtastic/firmware#3834] [Ideas]
### Platform

NRF52

### Description

Working on a tracker use case using rak4631 in a watch/e-ink device: https://openmeshwatch.github.io/

To save battery I'd like to have the GPS default to off, but be able to wake it and request a location broadcast over the mesh.

**Comment by HarukiToreda** [comment]
isn't that already possible with the Remote module?
